src/utils/configurationVault/getConfiguration.js

```diff
@@ -30,11 +30,11 @@ const getConfiguration = (): { get: Function, set: Function } => {
     const packageJson = `${cwd()}/package.json`
     const configurationFile = `${cwd()}/.gitmojirc.json`
 
-    if (pathExists.sync(packageJson) && require(packageJson)?.gitmoji) {
+    if (pathExistsSync(packageJson) && require(packageJson)?.gitmoji) {
       return require(packageJson).gitmoji
```

Comment:
We need to migrate this to `readFileSync` as `require` can't be used

Reply:
Completed this task in #928.